docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+3. Plug the USB stick into the Pi.
+4. Connect the 12V power supply to the brain board's Camcon connector.
+5. 3 Green LEDs on the Camcon end of the board will light.
+    - If they don't light immediately this board has failed, go to step 8.
+6. The 5 boot LEDs will light in order.
+    - If the first LED has not lit within 20 seconds this board has failed, go to step 8.
+    - If all 5 LEDs have not lit within approximately a minute this board has failed, go to step 8.
+7. The OK LED should then light a few colours before stabilising on green.
+    - If the LED stabilises on red this board has failed, go to step 8.
+    - If the LED has not lit within 20 seconds try replugging the USB stick. If it continues to not light this board has failed, go to step 8.
+8. Remove board making a separate pile for boards that failed.
+9. Repeat steps 2-8 for the next board.
+10. Copy the `tested_KCHs.txt` file off the USB stick. This is the KCH asset codes of all the passing brain boards.
+
+### Power Board
+
+Here we are testing all outputs can be enabled and current sense is functioning.
+Additionally, we test the buzzer, LEDs and start button are functioning and finally we test both the software and hardware undervoltage protection.
+
+For this test you will need:
+
+- The power board testing jig, pictured below
+- High current power supply (Dell DPS-750TB)
+- USB controllable power supply (Tenma 72-2545)
+- Power supply switcher
+
+![Power board jig](./img/power_jig.jpeg)
+<!-- TODO add picture of switcher connection -->
+
+!!! note
+
+    While testing undervoltage, errors will be printed about failing to connect to the board. This is expected behaviour.
+
+Test steps:
+
+1. Connect up the high current PSU and USB controlled PSU to the switcher circuit.
+2. Connect the switcher circuit to the test jig.
+3. Set the USB controlled PSU to 12V and 3A.
+4. Set all switches on the jig to up position.
+5. Set the on/off switch to on.
+6. Connect a USB cable to the board to test.
+7. Slide board into the jig from the top.
+8. Insert both sets of side plugs, ensuring they are fully inserted.
+9. Connect the board's XT60 connecter to the rear of the jig.
+10. Run the script and follow its instructions:<br>
+    `kit_test power_v4 --fw-ver 4.4.2 --test-uvlo --log power_testing.csv`
+11. Remove board making a separate pile for boards that failed.
+12. Repeat steps 6-9 and follow the script's instructions for the next board.
+
+### Motor Board
+
+Here we are testing the both motor outputs function and that the current sensing is functional.
+
+For this test you will need:
+
+- The motor board testing jig, pictured below
+- A power supply capable of 12 volts at 3 amps
+- 2x 4.7 Ohm power resistor with banana plugs
+
+![Motor board jig](./img/motor_jig.jpeg)
+
+Test steps:
+
+1. Unplug the motors from the jig.
+2. Connect the 4.7 Ohm resistors with banana plugs in place of the motors.
+3. Connect jig to PSU using banana leads.
+4. Set the PSU to 12V and 3A.
```

Comment:
Question: Does this need to be exactly 3A? For example, a high-current output on the power board is higher than this, but my understanding is still usable for the test?

Reply:
This is just the settings for the power supply. It gives enough current for the test to be run.